docs/src/pages/guides/page-layout.md

```diff
@@ -151,27 +139,62 @@ Things to consider:
 {{<Note>}}
 The frontmatter props `navOrder` and `order` have different functions.
 
-- `navOrder` identifies a top-level page and its position in the `TabList` across the top of the page.
-- `order` sets the order for all pages in a section that uses the `accordion` layout.
+- `navOrder` identifies a top-level page and its position in the `NavigationAccordion` across the top of the page.
+- `order` sets the order for all pages in a section that uses the `page` layout.
 
-For top-level pages using the `accordion` layout, besides setting `navOrder`, you will also set `order: 1` to make sure that the top-level page appears first in the [`NavigationAccordion`](/dr-ui/#navigationaccordion)
+For top-level pages using the `page` layout, besides setting `navOrder`, you will also set `order: 1` to make sure that the top-level page appears first in the [`NavigationAccordion`](/dr-ui/#navigationaccordion)
 {{</Note>}}
 
-## Custom sidebars
+## Overview header
+
+The main page for each docs site displays the [`OverviewHeader`](/dr-ui/#overviewheader) to orient the user to the product.
+
+To add `OverviewHeader` to your page, pass the props of the component in the frontMatter under `overviewHeader`.
+
+```yaml
+overviewHeader:
+  title: 'Dr. UI'
+  features:
+    - 'React components to build documentation sites'
+    - 'Support for IE 11 and all modern browsers'
+  changelogLink: /dr-ui/changelog/
+  installLink: https://github.com/mapbox/dr-ui/blob/main/README.md
+  ghLink: https://github.com/mapbox/dr-ui
+```
+
+To include dynamic variables, such as `version` or use the `AppropriateImage` component for the value of `image`, update the `frontMatter` prop in site's `PageLayout` component (found in `page-shell.js`).
+
+The following example is the value of the `frontMatter` prop which overrides the values of `version` and `image` in the `overviewHeader` frontMatter prop object:
+
+```js
+{
+  ...frontMatter,
+  ...frontMatter.overviewHeader && {
+      overviewHeader: {
+        ...frontMatter.overviewHeader,
+        version: myVariable,
+        image: <AppropriateImage id="documentation-astronaut" />
+      }
+    }
+  })
+}
+```
+
+This technique may also be used by multi-structured sites, although you'll need to add logic to help determine which version constant to use.
+
+## Custom aside
 
-Sites like Mapbox GL JS require custom sidebars since the data is derived from multiple sources. Similarly, the [Dr. UI component's page](/dr-ui/) also uses a custom sidebar, see the example below for how to conditionally display a custom sidebar:
+Sites like Mapbox GL JS require custom asides since the data is derived from multiple sources. Similarly, the [Dr. UI component's page](/dr-ui/) also uses a custom aside, see the example below for how to conditionally display a custom sidebar:
```

Comment:
```suggestion
Sites like Mapbox GL JS require custom asides since the data is derived from multiple sources. Similarly, the [Dr. UI components page](/dr-ui/) also uses a custom aside, see the example below for how to conditionally display a custom sidebar:
```